frontend/src/components/table/TableItem.tsx

```diff
@@ -89,8 +91,15 @@ export const TableItem = ({
     <AccordionItem value={user.user_id}>
       <AccordionTrigger className="relative flex cursor-pointer justify-between gap-x-6 px-4 py-5 hover:bg-gray-50 sm:px-6">
         <div className="flex items-center gap-x-2">
-          <span className="flex h-12 w-12 items-center justify-center rounded-full bg-indigo-100 align-middle text-4xl">
-            {textToEmoji(user.first_name + user.last_name)}
+          <span className="flex h-12 w-12 pb-1 items-center justify-center rounded-full bg-indigo-100 align-middle text-3xl text-[#4C4C51]">
+            {/* Displays the ith placement on the leaderboard if i is defined, otherwise displays an emoji */}
+            {i !== undefined
```

Comment:
`Empty string`, `null` and `undefined` are valued as `false` in a if statement. So you can just write `i ? "true" : "false"` instead.

```suggestion
           { i
```

Reply:
This wouldn't work because then placement 0 (placement 1) would be skiped